docs/tech-reference/plugin_conf.rst

```diff
@@ -85,16 +85,16 @@ Install Distributor
 Type ID: ``puppet_install_distributor``
 
 This distributor publishes modules by actually installing them into a given
-``install_path`` on the Pulp server's filesystem. The use case is that you want
+``install_path/subdir`` on the Pulp server's filesystem. This is useful when you want
```

Comment:
Since subdir is optional, would it be useful to do something like ``install_path[/subdir]``, and highlight its optional-ness here, and in other places where it occurs?